versions/3.0.md

```diff
@@ -20,51 +20,51 @@ Additional utilities can also take advantage of the resulting files, such as tes
-		- [Contact Object](#contactObject)
-		- [License Object](#licenseObject)
-		- [Server Object](#serverObject)
+  - [Format](#format)
+  - [File Structure](#fileStructure)
+  - [Data Types](#dataTypes)
+  - [Relative References In URLs](#relativeReferences)
+  - [Schema](#schema)
+    - [OpenAPI Object](#oasObject)
+    - [Info Object](#infoObject)
+    - [Contact Object](#contactObject)
+    - [License Object](#licenseObject)
+    - [Server Object](#serverObject)
         - [Server Variables Object](#serverVariablesObject)
         - [Server Variable Object](#serverVariableObject)
-		- [Components Object](#componentsObject)
-		- [Paths Object](#pathsObject)
-		- [Path Item Object](#pathItemObject)
-		- [Operation Object](#operationObject)
-		- [External Documentation Object](#externalDocumentationObject)
-		- [Parameter Object](#parameterObject)
-		- [Request Body Object](#requestBodyObject)
-		- [Content Object](#contentObject)
-		- [Media Type Object](#mediaTypeObject)
-		- [Responses Object](#responsesObject)
-		- [Response Object](#responseObject)
-		- [Headers Object](#headersObject)
-		- [Example Object](#exampleObject)
-		- [Links Object](#linksObject)
-		- [Link Object](#linkObject)
-		- [Link Parameters Object](#linkParametersObject)
-		- [Header Object](#headerObject)
-		- [Tag Object](#tagObject)
-		- [Examples Object](#examplesObject)
-		- [Reference Object](#referenceObject)
-		- [Schema Object](#schemaObject)
-		- [XML Object](#xmlObject)
-		- [Security Scheme Object](#securitySchemeObject)
-		- [Scopes Object](#scopesObject)
-		- [Security Requirement Object](#securityRequirementObject)
-	- [Specification Extensions](#specificationExtensions)
-	- [Security Filtering](#securityFiltering)
+    - [Components Object](#componentsObject)
+    - [Paths Object](#pathsObject)
+    - [Path Item Object](#pathItemObject)
+    - [Operation Object](#operationObject)
+    - [External Documentation Object](#externalDocumentationObject)
+    - [Parameter Object](#parameterObject)
+    - [Request Body Object](#requestBodyObject)
+    - [Content Object](#contentObject)
+    - [Media Type Object](#mediaTypeObject)
+    - [Responses Object](#responsesObject)
+    - [Response Object](#responseObject)
+    - [Headers Object](#headersObject)
+    - [Example Object](#exampleObject)
+    - [Links Object](#linksObject)
+    - [Link Object](#linkObject)
+    - [Link Parameters Object](#linkParametersObject)
+    - [Header Object](#headerObject)
+    - [Tag Object](#tagObject)
+    - [Examples Object](#examplesObject)
+    - [Reference Object](#referenceObject)
+    - [Schema Object](#schemaObject)
+    - [XML Object](#xmlObject)
+    - [Security Scheme Object](#securitySchemeObject)
+    - [Scopes Object](#scopesObject)
+    - [Security Requirement Object](#securityRequirementObject)
+  - [Specification Extensions](#specificationExtensions)
+  - [Security Filtering](#securityFiltering)
 - [Appendix A: Revision History](#revisionHistory)
-	
+
 
```

Comment:
That was unintentional. The problem is that most of Table of Contents is tab spaced, but server variable / server variables is using spaces. Odds are whomever put those two items in were trying to get the list to line up properly and the were using spaces where the rest of the list is using tabs. I'll fix that up also.